templates/cassiopeia/scss/template-rtl.scss

```
@@ -40,6 +40,11 @@ dd {
   padding-right: 0;
 }
 
+.metismenu .mm-toggler {
+  margin-right: .5em;
+  margin-left: 0;
```

```suggestion
  margin-left: 0 !important;
```
Without important, the marging between the dropdown toggle and the next menu item is too big.